docs/shared/recipes/nx-release/getting-started-with-nx-release.md

```diff
@@ -0,0 +1,277 @@
+# Getting Started with Nx Release
+
+This recipe guides you through versioning packages, generating changelogs, and publishing packages in a JavaScript monorepo with Nx Release.
+
+## Getting set up
+
+### Installing Nx
+
+Ensure that Nx is installed in your monorepo. Check out the [Installation docs](/getting-started/installation) for instructions on created a new Nx workspace or adding Nx to an existing project.
+
+### Adding the JavaScript Plugin
+
+The @nx/js package is required for Nx Release to manage and release JavaScript packages. Add it if it is not already installed:
+
+```shell
+npm install -D @nx/js
```

Comment:
```suggestion
nx add @nx/js
```

Reply:
We're using `nx add` in Nx 18+ because it will autodetect the correct version to install and run the `init` generator.